services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusService.kt

```diff
@@ -649,23 +653,7 @@ class HakemusService(
                     JohtoselvityshakemusPdfEncoder.createPdf(data, totalArea, areas, attachments)
                 }
                 is KaivuilmoitusData -> {
-                    val areas =
-                        data.areas?.associate {
-                            it.hankealueId to
-                                geometriatDao.calculateCombinedArea(
-                                    it.tyoalueet.map { alue -> alue.geometry }
-                                )
-                        } ?: mapOf()
-                    val hankealueet = hankeRepository.findByHankeTunnus(hankeTunnus)?.alueet
-                    val hankealueNames = hankealueet?.associate { it.id to it.nimi } ?: mapOf()
-                    val alueet =
-                        data.areas?.map {
-                            EnrichedKaivuilmoitusalue(
-                                areas[it.hankealueId],
-                                hankealueNames[it.hankealueId] ?: "Nimetön hankealue",
-                                it,
-                            )
-                        }
+                    val alueet = data.areas?.let { enrichKaivuilmoitusalueet(it, hankeTunnus) }
```

Comment:
This is now named `alueet` but for johtoselvitys the similar name is still `areas`. Would it be better to have both named as `alueet`?

Reply:
Well, actually, `areas` in for johtoselvitys means the geometrical area, so, nevermind. It is better to have different names as they are different concepts.